src/helpers/augmentedReality/storageNameCreator.js

```diff
@@ -1,6 +1,13 @@
+import { documentDirectory } from 'expo-file-system';
+
 export const storageNameCreator = ({ dataItem, objectItem }) => {
-  const { id: objectId } = dataItem;
-  const { title, type, id } = objectItem;
+  const { id: objectId, title: objectTitle } = dataItem;
+  const { title, type } = objectItem;
+  const replacedObjectTitle = objectTitle.replace(/\s+/g, '');
 
-  return `${objectId}-${title}${id}.${type}`;
+  return {
+    directoryName: documentDirectory + `${replacedObjectTitle}_${objectId}/${title}.${type}`,
+    folderName: documentDirectory + `${replacedObjectTitle}_${objectId}`,
+    storageName: `${replacedObjectTitle}_${objectId}_${title}.${type}`
```

Comment:
why did the filename divider become underscore instead of hyphens?

Reply:
No particular reason 😂🤷‍♂️